onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_execution_provider.cc

```diff
@@ -20,17 +22,44 @@
 
 namespace onnxruntime {
 
+namespace {
+
 constexpr const char* NNAPI = "Nnapi";
 
+namespace nnapi_env_vars {
+// Specifies a list of stop op types.
+// Nodes of a type in the stop op types and nodes downstream from them will not be run by the NNAPI EP.
+// The value should be a ","-delimited list of op types. For example, "Add,Sub".
+// An empty value is interpreted as specifying the default set of stop op types.
+// To specify an empty stop ops types list and disable stop op exclusion, set the value to ",".
+constexpr const char* kPartitioningStopOps = "ORT_NNAPI_PARTITIONING_STOP_OPS";
+}  // namespace nnapi_env_vars
+
 constexpr std::array kDefaultPartitioningStopOps{
     "NonMaxSuppression",
 };
 
+constexpr const char* kPartitioningStopOpsListDelimiter = ",";
+
+std::unordered_set<std::string> GetPartitioningStopOps() {
+  const auto stop_ops_list_value = Env::Default().GetEnvironmentVar(nnapi_env_vars::kPartitioningStopOps);
```

Comment:
I am not sure if setting the environment variable is a good way to pass settings on Android, especially those set in java and pass down to native code. We can try it in an Android App to see if it works.
Maybe we can try to use session config entries for this